pkg/reconciler/testing/trustedresources.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
```

Comment:
Let's move this into `pkg/reconciler/internal/testing` - this will remove any funcs from the public API surface.

Reply:
Yup!

Reply:
created a new path and moved there